### ISSUE TITLE
[feature request] Allow for specifying number padding to prevent menu icons shift

### ISSUE THREAD

**Issue by ElSamhaa** [gao-sun/eul#261]
Whenever the stats numbers change from a single digit to multiple digit, the menu icons after the changed eul icon shift left and vice versa.

https://github.com/gao-sun/eul/assets/34912861/223b42f5-8d87-4fd7-bab2-ad06f534418c

**Comment by 54ycf** [comment]
I agree with you! I think it's better to allocate a constant space for each statistic, ensuring that changes in digit numbers don't affect the width of icon. I hope the maintainer can consider this issue.
